Objects/typeobject.c

```diff
@@ -9906,8 +9906,14 @@ SLOT1(slot_nb_inplace_remainder, __imod__, PyObject *)
 static PyObject *
 slot_nb_inplace_power(PyObject *self, PyObject * arg1, PyObject *arg2)
 {
-    PyObject *stack[2] = {self, arg1};
-    return vectorcall_method(&_Py_ID(__ipow__), stack, 2);
+    if (1||arg2 == Py_None) {
+        PyObject *stack[2] = {self, arg1};
+        return vectorcall_method(&_Py_ID(__ipow__), stack, 2);
+    }
+    else {
+        PyObject *stack[3] = {self, arg1, arg2};
+        return vectorcall_method(&_Py_ID(__ipow__), stack, 3);
```

Comment:
This still untested.

Reply:
Oh, this change was added unintentionally in this PR.